package/boot/uboot-envtools/files/ramips

```diff
@@ -30,6 +30,7 @@ wsr-600|\
 zbt-wg2626)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
 	;;
+mir3p|\
```

Comment:
ramips should support taking the boardname from the machine compatible in the DTS

so, it would be better to use "xiaomi,mir3p" instead of mir3p in this and the following cases.

Reply:
Yeah. Sorry, after I put in my pull request I saw that someone had made very similar suggestions to the last person to add a ramips device... My bad for not having seen that sooner. Fixed.